### ISSUE TITLE
Closed channel, sats transferred back to OnChain confirmed over 100 blocks ago but still show "Unconfirmed" in RTL

### ISSUE THREAD

**Issue by davterra** [Ride-The-Lightning/RTL#399]
Using RTL 0.8.3

Closed a channel yesterday and resulting fund in on-chain wallet still show as unconfirmed. A check of the txid shows the tx confirmed yesterday. I have done rescanblockchain for the relevant time interval on my attached bitcoind to make sure it is current.
Tried restarting RTL and clearing browser cache as well.

**Comment by ShahanaFarooqui** [comment]
@davterra Can you please specify your node’s lnImplementation? Is it LND/CTL/ECL?

**Comment by davterra** [comment]
> @davterra Can you please specify your node’s lnImplementation? Is it LND/CTL/ECL?

Yes, sorry to have forgotten:
LND 0.10.0-beta.rc5
BTC Core 0.20.0

**Comment by saubyk** [comment]
Hi @davterra, the unconfirmed balance that you are seeing on RTL is what is being returned by the LND API.
Try restarting LND once.

Also, was the channel closure forced or cooperative?

**Comment by davterra** [comment]
The channel close was cooperative.

This BTC/LND/RTL implementation is one of two that I have. The other is always-on, but this one gets shut down for a few hours, or up to a couple of days now and then. It has been shut down since this morning. When I restarted the nodes, I saw that the BTC in question now shows as confirmed. 

If this happens again, I will restart LND and report back to you the results.

Thanks for your prompt assistance.

If I dont hear otherwise from you in the next day, I'll close this issue.
